scripts/utils.ts

```diff
@@ -1,6 +1,24 @@
 import { promises as filesystem } from 'fs'
 import * as path from 'path'
 import { CompilerOutput, CompilerInput, compileStandardWrapper } from 'solc'
+import { SIGNER } from './constants'
+
+export enum ScriptErrorCode {
+	UNKNOWN_ERROR = 100,
+	RPC_URL_NOT_FOUND,
+	FACTORY_ALREADY_DEPLOYED,
+	CHAIN_NOT_LISTED,
+	FACTORY_DIFFERENT_BYTECODE,
+	FACTORY_PRE_DEPLOYED,
+	FACTORY_NOT_ADDED_TO_REPO,
+	FACTORY_DEPLOYER_ACCOUNT_NONCE_BURNED,
+	GAS_PRICE_NOT_RETRIEVED,
+	GAS_LIMIT_NOT_ESTIMATED,
+	GAS_LIMIT_ESTIMATION_FAILED,
+	DEPLOYMENT_SIMULATION_FAILED,
+	FACTORY_DEPLOYMENT_SIMULATION_DIFFERENT_BYTECODE,
+	PREFUND_NEEDED
+}
```

Comment:
These error codes are specific to `new-chain.ts` and should be moved there.

Reply:
Moved 👍🏾